### PR TITLE
Fix problems with python 2 and 3

### PR DESCRIPTION
**Pull request by patkan**
This should fix most python2/3-problems.

Since this change is a bit larger I will keep this PR open a bit before merging.

### CODE REVIEW

File: escpos/printer.py [hunk 1]
```diff
@@ -218,11 +226,12 @@ def _raw(self, msg):
...
         :param msg: arbitrary code to be printed
+        :type msg: bytes
         """
-        if type(msg) is str:
-            self.device.write(msg.encode())
-        else:
-            self.device.write(msg)
+        #if type(msg) is str:
+        #    self.device.write(msg.encode())
+        #else:
```
Comment (braveheuel) [inline escpos/printer.py]: I think it is better to remove the code, not comment it out

Comment (patkan) [inline escpos/printer.py]: You are totally right.

**Comment by patkan** [comment]
I think this is ready for merging, OK?

**Comment by Loafdude** [comment]
I haven't had any issues running it on python 2.7.x

**Pull request merged**